services/transaction_body.proto

```diff
@@ -375,5 +376,10 @@ message TransactionBody {
      * A transaction body for a `deleteNode` request.
      */
     NodeDeleteTransactionBody nodeDelete = 56;
+
+    /**
+     * A transaction body for a `rejectToken` request.
```

Comment:
This specification comment is not particularly helpful; it merely restates the type of a variable in more words.
A better option (which may still not be perfect, but is much more useful):
```suggestion
     * A transaction body to "reject" undesired tokens.<br/>
     * This transaction will transfer one or more tokens or token
     * balances held by the requesting account to the treasury
     * for each token type.
     * <p>
     * Each transfer MUST be one of the following:
     * <ul>
     *   <li>A single non-fungible/unique token.</li>
     *   <li>The full balance held for a fungible/common
     *       token type.</li>
     * </ul>
     * When complete, the requesting account SHALL NOT hold the
     * rejected tokens.<br/>
     * Custom fees and royalties defined for the tokens rejected
     * SHALL NOT be charged for this transaction.
```

Reply:
I've adjusted it, but I haven't included this whole comment, as the whole description already exists in the token_reject.proto itself, and all the comments for other transaction bodies in transaction_body.proto are short and concise.

If you think it's still better to have the full description here and also in the TokenRejectTransactionBody itself (token_reject.proto), I could add it. 🙂

Reply:
We should have the more full description.  The short (minimal, really) descriptions currently present in most cases in this file are all scheduled to be replaced with thorough _specification_.

Reply:
Got it. I've adjusted it.